test/jasmine/tests/svg_text_utils_test.js

```diff
@@ -301,7 +301,8 @@ describe('svg+text utils', function() {
             var node = mockTextSVGElement('SO<sub>4<br>44</sub>');
             expect(node.html()).toBe(
                 '<tspan class="line" dy="0em">SO​' +
-                    '<tspan style="font-size:70%" dy="0.3em">4</tspan></tspan>' +
+                    '<tspan style="font-size:70%" dy="0.3em">4</tspan>' +
+                    '<tspan dy="-0.21em">​</tspan></tspan>' +
```

Comment:
I don't think this has any visual consequences, but `<sup>`/`<sub>` tags previously didn't get their proper reset elements when they spanned line breaks.